### PR TITLE
changed translation for "nuclear" into the better german translation

### PR DESCRIPTION
**Pull request by Spanni26**

### CODE REVIEW

File: web/locales/de.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
         "consumption": "Verbrauch",
         "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie",
         "selectLanguage": "Sprache auswählen",
-        "lowCarbDescription": "Beinhaltet regenerative Energie and Atomkraft"
+        "lowCarbDescription": "Beinhaltet regenerative Energie and Kernenergie"
```
Comment (alixunderplatz) [inline web/locales/de.json]: two old typos could be fixed in the same run:
"Beinhaltet regenerative Energie***n***  ***u***nd Kernenergie"

**Comment by Spanni26** [comment]
Atomkraft is more negative (reference to atombombs) and technically wrong.  No atoms are split. So I think Kernenergie is more correct.Am 22.12.2019 20:55 schrieb Alex <notifications@github.com>:@alixunderplatz commented on this pull request.



hi @Spanni26, thank you for your contribution!

"Atomkraft" and "Kernenergie" are synonyms in the German language and both are broadly used for nuclear power. I don't see a real advantage in changing it from one to the other.

https://de.wikipedia.org/wiki/Kernenergie



In web/locales/de.json:

> @@ -93,7 +93,7 @@

         "consumption": "Verbrauch",

         "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie",

         "selectLanguage": "Sprache auswählen",

-        "lowCarbDescription": "Beinhaltet regenerative Energie and Atomkraft"

+        "lowCarbDescription": "Beinhaltet regenerative Energie and Kernenergie"



two old typos could be fixed in the same run:

"Beinhaltet regenerative Energien  und Kernenergie"

File: web/locales/de.json (hunk 1 above)
Comment (jarek) [inline web/locales/de.json]:
```suggestion
        "lowCarbDescription": "Beinhaltet regenerative Energien und Kernenergie"
```

**Comment by corradio** [comment]
Closing due to inactivity.

**Pull request closed without merging**